### PR TITLE
Add new troubleshooting section to Operational Procedures.

### PR DESCRIPTION
**Pull request by tribeiro**
Add troubleshoot procedure for components in offline.

### CODE REVIEW

File: Operational-Procedures/Troubleshooting/no-historical-data.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.. |author| replace:: *Tiago Ribeiro*
+.. |contributors| replace:: *Michael Reuter, Patrick Ingraham*
+
+.. _troubleshooting-no-historical-data:
+
+##################
+No Historical Data
+##################
+
+This issue usually manifest by a failure to perform state transitions (enable a component or send them to STANDBY) using the higher level software.
```
Comment (isotuela) [inline Operational-Procedures/Troubleshooting/no-historical-data.rst:10]: **manifests

File: Operational-Procedures/Troubleshooting/no-historical-data.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+This issue affects the Rubin Observatory Control System because the way the middleware message system (DDS) works.
+
+For more details see the `observatory control`_ documentation page.
+
+.. _observatory control: https://obs-controls.lsst.io
+
+To resolve this issue we can manually perform the state transition by sending the command directly to the CSC.
+The easiest way to do this is using the LOVE Summary state command feature.
+
+The state transition works as follows:
```
Comment (isotuela) [inline Operational-Procedures/Troubleshooting/no-historical-data.rst:32]: I think a diagram with the allowed state transitions would be cool!

**Review by isotuela — COMMENTED** [review]
Thanks for writing these.

File: Operational-Procedures/Troubleshooting/component-offline.rst [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+.. _troubleshooting-component-offline:
+
+#################
+Component Offline
+#################
+
+
+Most (but not all) components in the Rubin Observatory Control System, when sent to OFFLINE will terminate its execution process, and will no longer be available in the system.
+In order to restore them, it is then necessary to restart the process that runs the components.
```
[the remark this answers is not in the document]
Comment (tribeiro) [reply Operational-Procedures/Troubleshooting/component-offline.rst:11]: If there is a power glitch or something the CSC will come back in STANDBY.. the only way a CSC goes to offline is when someone sends an `exitControl` command. I'll clarify.

File: Operational-Procedures/Troubleshooting/component-offline.rst [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+    :name: fig-love-mtptg-offline
+
+    LOVE Summary State view showing the MTPtg in OFFLINE state with no heartbeats, indicating the CSC is no longer running.
+
+For more details about what is behind this behavior see the `observatory control`_ documentation page.
+
+.. _observatory control: https://obs-controls.lsst.io
+
+Most components on Rubin system runs as containers on a Kubernetes cluster.
+Therefore, most-likely, when restarting a component after it was sent to OFFLINE state, requires restarting the process on the Kubernetes cluster.
```
[the remark this answers is not in the document]
Comment (tribeiro) [reply Operational-Procedures/Troubleshooting/component-offline.rst:25]: I changed the first occurrence to "restoring"

File: Operational-Procedures/Troubleshooting/component-offline.rst [hunk 5]
```diff
@@ -0,0 +1,126 @@
+.. |author| replace:: *Tiago Ribeiro*
+.. |contributors| replace:: *Michael Reuter, Patrick Ingraham*
+
+.. _troubleshooting-component-offline:
+
+#################
+Component Offline
```
[the remark this answers is not in the document]
Comment (Slimleashma) [reply Operational-Procedures/Troubleshooting/component-offline.rst:7]: I would agree with Ioana's comment - the title is vague.

File: Operational-Procedures/Troubleshooting/component-offline.rst [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+#################
+
+.. warning::
+
+    This procedure contains steps that may affect the system in crical ways.
```
Comment (Slimleashma) [inline Operational-Procedures/Troubleshooting/component-offline.rst:12]: *critical

File: Operational-Procedures/Troubleshooting/component-offline.rst [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+
+    If you are unsure about any of the steps described here, contact support personnel.
+
+
+* Most (but not all) components in the Rubin Observatory Control System, when sent to OFFLINE will terminate its execution process and will no longer be available in the system.
```
Comment (Slimleashma) [inline Operational-Procedures/Troubleshooting/component-offline.rst:17]: Change to: * Most components in the Rubin Observatory Control System, when sent to OFFLINE, will terminate its execution process and will no longer be available in the system.

File: Operational-Procedures/Troubleshooting/component-offline.rst [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+
+* This condition only happens if someone sends the ``exitControl`` command to the CSC.
+
+  Sending the ``exitControl`` command to a CSC is regularly done when one wants to update a version of the CSC or needs to restart the process to recover from some fault conditions.
+
+* A reduced number of CSCs will continue to run after receiving ``exitControl``.
+
+  At the time of this writting, only the camera systems have this behavior.
```
Comment (Slimleashma) [inline Operational-Procedures/Troubleshooting/component-offline.rst:25]: Does this mean that the camera systems will continue to run after receiving an  "exit control" command?

File: Operational-Procedures/Troubleshooting/component-offline.rst [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+
+Restoring a CSC that exited after responding to ``exitControl`` requires one to restart the process that runs it.
```
Comment (Slimleashma) [inline Operational-Procedures/Troubleshooting/component-offline.rst:27]: Change to: "Restoring a CSC after sending an ``exitControl`` requires one to restart the process that runs it.

File: Operational-Procedures/Troubleshooting/component-offline.rst (hunk 8 above)
Comment (tribeiro) [reply Operational-Procedures/Troubleshooting/component-offline.rst:25]: correct

File: Operational-Procedures/Troubleshooting/component-offline.rst [hunk 10]
```diff
@@ -0,0 +1,148 @@
...
+    :name: fig-love-mtptg-offline
+
+    LOVE Summary State view showing the MTPtg in OFFLINE state with no heartbeats, indicating the CSC is no longer running.
+
+For more details about what is behind this behavior see the `observatory control`_ documentation page.
+
+.. _observatory control: https://obs-controls.lsst.io
+
+Most components on Rubin system runs as containers on a Kubernetes cluster.
+Therefore, most-likely, when restoring a component after it was sent to OFFLINE state, requires restarting the process on the Kubernetes cluster.
```
Comment (Slimleashma) [inline Operational-Procedures/Troubleshooting/component-offline.rst:49]: Change to: Therefore, it is likely that restoring a component after it was sent to OFFLINE state requires restarting the process on the Kubernetes cluster.

File: Operational-Procedures/Troubleshooting/component-offline.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (tribeiro) [reply Operational-Procedures/Troubleshooting/component-offline.rst:7]: yes! of course! Slipped through the cracks.. sorry

File: Operational-Procedures/Troubleshooting/no-historical-data.rst [hunk 11]
```diff
@@ -0,0 +1,61 @@
+.. |author| replace:: *Tiago Ribeiro*
+.. |contributors| replace:: *Michael Reuter, Patrick Ingraham*
+
+.. _troubleshooting-no-historical-data:
+
+##################
+No Historical Data
+##################
+
+This issue usually manifests by a failure to perform state transitions (enable a component or send them to STANDBY) using the higher level software.
```
Comment (Slimleashma) [inline Operational-Procedures/Troubleshooting/no-historical-data.rst:10]: change enable to ENABLE

**Review by Slimleashma — APPROVED** [review]
These look good + addressing Ioana's comments

File: Operational-Procedures/Troubleshooting/component-offline.rst (hunk 8 above)
Comment (Slimleashma) [reply Operational-Procedures/Troubleshooting/component-offline.rst:25]: Can you state that more clearly?

Comment (tribeiro) [reply Operational-Procedures/Troubleshooting/component-offline.rst:25]: done

**Pull request merged**